### PR TITLE
upgrade mustermann-grape to 1.0.0

### PR DESCRIPTION
**Pull request by namusyaka**
ref #1511

### CODE REVIEW

**Comment by dblock** [comment]
That's a very rigid dependency and prevents us from releasing Grape v. next. Would `~> 1.0.0` include the beta?

**Comment by dblock** [comment]
I would also add a CHANGELOG line that says that #1511 is fixed by this PR.

**Comment by namusyaka** [comment]
>Would ~> 1.0.0 include the beta?

Nope, `~> 1.0.0` does not include beta release.
But maybe we don't have to unify versioning with official mustermann. What do you think?

>I would also add a CHANGELOG line that says that #1511 is fixed by this PR.

Fixed.

**Comment by dblock** [comment]
I don't think unifying versioning with mustermann is needed, but otherwise I think you should release 1.0 (not a beta) and call it a day. I don't think we should merge this PR until Grape is forward compatible.

**Comment by namusyaka** [comment]
I got your point, it's reasonable.
Okay, I'm going to confirm the plan for releasing mustermann-1.0.0.

**Comment by namusyaka** [comment]
@dblock mustermann-1.0.0 has been released, so I cut a new release of mustermann-grape-1.0.0.

File: grape.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.license     = 'MIT'
 
   s.add_runtime_dependency 'rack', '>= 1.3.0'
-  s.add_runtime_dependency 'mustermann-grape', '~> 0.4.0'
+  s.add_runtime_dependency 'mustermann-grape', '1.0.0'
```
Comment (dblock) [inline grape.gemspec]: Should be `~> 1.0.0` or it will lock at 1.0 no?

Comment (namusyaka) [inline grape.gemspec]: Right. Fixed.

**Pull request merged**